app/components/organizations_index/organizations_index.erb

```diff
@@ -0,0 +1,22 @@
+<%= c 'main', style: :lightestGrayBackground, **attrs do %>
+  <%= c 'page_header', styles: [:flexboxColumn, :wide, :xlargePaddingTop] do %>
+    <%= c 'heading' do %>
+      <%= t('.main_heading', first_name: current_user.first_name) %>
+    <% end %>
+    <%= c 'heading', tag: :h2 do %>
+      <%= t('.heading') %>
+    <% end %>
+  <% end %>
+
+  <%= c 'section', styles: [:wide, :noSpaceBetween] do %>
+    <ul class="OrganizationsIndex-organizationsList">
+      <% organizations.each do |organization| %>
+        <li>
+          <%= c 'button', link: organization_dashboard_path(organization), styles: [:plain, :underline] do %>
+            <%= organization.name %>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+<% end %>
```

Comment:
@soey As we've discussed, I'm not really a fan of having a `_index` component simply for the fact of adding a css to the component level structure, as we do here.
I could go through my old PRs where I introduced adding css to views and we could change this here, but honestly, I think it's not related to this PR.
We could discuss this pattern and change it for the contributors_index as well in another PR. I'm not sure if there are other places that follow this pattern.

Reply:
I very much like the idea to KISS